nsv13/code/datums/mood_events/nsv_events.dm

```diff
@@ -1,6 +1,6 @@
 /datum/mood_event/moth_drink_blood
 	description = "<span class='nicegreen'>That hit the spot!</span>\n"
-	mood_change = 10
+	mood_change = 10 //Why is a moth drinking blood a mood buff the level of being a cultist?? And for *ten minutes*???
 	timeout = 10 MINUTES
```

Comment:
```suggestion
	mood_change = 3
	timeout = 7 MINUTES
```
You can also just suggest a change, code comments aren't really the place for complaints.

Reply:
I usually leave these sorts of comments as pointer to myself or others to change this in the future so it doesn't go forgotten, since sometimes I *do* manage to contain the urge to resolve every single bit of misbehavior in the same PR.

Reply:
I think the new values I chose are more fitting for the type of moodlet it is (based on other drink related buffs). I'm fine with just having that in this PR since it's not too big of a thing. I can also open it as a new PR but then please remove the comment to avoid a conflict.